CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## v1.9
 
+### Compatibility with Rematch 5
+
+Together with patch 10.2 Gello releases Rematch 5, which is a major change both under the hood and in the user interface. To properly integrate again we had to make changes to this addon as well. To allow for users updating with a delay, this addon is compatible with Rematch 4 and Rematch 5 at the same time.
+
+Rematch offers `/rematch reset everything` in case anything goes wrong in the update process, which this addon hooks to also save your matching scripts and tries to restore them in a reset case as well. Hopefully, there is no reason to use it though.
+
+One small change you will encounter is that the "script button" in the team list is now non-interactive: It only shows whether there is a script or not, but you can't click it anymore to edit the script. The option to create and edit the script from the right click menu is still there, of course.
+
```

Comment:
Do users need to do anything else other then just update rematch and then PBS?

Reply:
They don't, and the order is irrelevant.